requirements.txt

```diff
@@ -6,3 +6,4 @@ tensorboard
 tensorboardX
 typeguard
 textgrid
+onnxruntime-gpu
```

Comment:
Ask user to install it in export_onnx.py if the package is too big.

Reply:
Got it.